nibabel/testing/__init__.py

```diff
@@ -59,6 +60,17 @@ def assert_allclose_safely(a, b, match_nans=True):
     assert_true(np.allclose(a, b))
 
 
+def assert_re_in(regex, c, flags=0):
+    """Assert that container (list, str, etc) contains entry matching the regex
+    """
```

Comment:
this one came from datalad (my copyright/license, so no problem there), so I decided just to keep it as is ;)